packages/react/src/components/Tag/Tag-test.js

```diff
@@ -6,87 +6,59 @@
  */
 
 import React from 'react';
-import Tag from '../Tag';
-import TagSkeleton from '../Tag/Tag.Skeleton';
+import { Tag } from 'carbon-components-react';
 import { Add16 } from '@carbon/icons-react';
-import { shallow } from 'enzyme';
-import { settings } from 'carbon-components';
-import { render, cleanup } from '@carbon/test-utils/react';
-
-const { prefix } = settings;
+import { render, screen } from '@testing-library/react';
 
 describe('Tag', () => {
-  afterEach(cleanup);
-
   describe('automated accessibility testing', () => {
     it('should have no Axe violations', async () => {
-      const { container } = render(<Tag type="red">This is not a tag</Tag>);
+      const { container } = render(<Tag type="red">Dog</Tag>);
```

Comment:
```suggestion
      const { container } = render(<Tag type="red">test-tag</Tag>);
```

For test labels we should just stick with either `test` or `test-<noun>`, I think, otherwise we end up with all sorts of random strings in examples lol